web/src/components/KebabMenu.tsx

```diff
@@ -95,8 +102,8 @@ const KebabMenu = ({ items, show, setShow, Footer }: KebabMenuProps): ReactEleme
         className='kebab-menu'
         show={show}
         style={{
-          visibility: show ? 'visible' : 'hidden',
-          top: window.scrollY > 0 ? `${window.scrollY}px` : undefined,
+          display: show ? 'block' : 'none',
+          top: scrollY > 0 ? `${scrollY}px` : undefined,
```

Comment:
:thought_balloon:  I tried to remove this top completely and it didn't affect anything at this page not sure if we can remove it...

Reply:
fair question,I think we do not need it here completely

Reply:
@f1sh1918 do you still remember why we needed this in the first place? To me it also looks like it works just fine without setting anything as `top`.

Reply:
this is required!
This ensures that the content of the kebab menu will be opened on top position even if you scroll down the page and then open the kebab menu

<img width="205" alt="image" src="https://github.com/user-attachments/assets/6e67d9c0-8c20-4351-a6c4-2c84bbde2a67" />

Reply:
Oh, very true... I thought I tested that, but seems like I didn't. Well, then please keep this @lunars97 :) I think we are good to merge this as it is then!

Reply:
🙈